servers/visual/visual_server_scene.cpp

```diff
@@ -2365,7 +2380,21 @@ void VisualServerScene::render_camera(RID p_camera, RID p_scenario, Size2 p_view
 					camera->zfar,
 					camera->vaspect);
 			ortho = false;
-
+		} break;
+		case Camera::OBLIQUE: {
+			camera_matrix.set_oblique(
+					camera->fov,
+					p_viewport_size.width / (float)p_viewport_size.height,
+					camera->znear,
+					camera->zfar,
+					Quat(/* clang-format off */
+						camera->oblique_plane.normal.x,
+						camera->oblique_plane.normal.y,
+						camera->oblique_plane.normal.z,
+						camera->oblique_plane.d
+					), /* clang-format on */
```

Comment:
Instead of tweaking `clang-format`, I think you can make the `Quat` a local variable here and trust the compiler to optimize it out.